docs/catalog.md

```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
   }
   ```
 
+### Using an action sequence and a change trigger to get a document from a Cloudant database
```

Comment:
get -> process?
it's not clear why i'd need a sequence just to get the document - it's really only necessary to operate on the document.